Source/Core/Core/Config/MainSettings.cpp

```diff
@@ -179,6 +180,9 @@ const Info<u32> MAIN_MEM1_SIZE{{System::Main, "Core", "MEM1Size"}, Memory::MEM1_
 const Info<u32> MAIN_MEM2_SIZE{{System::Main, "Core", "MEM2Size"}, Memory::MEM2_SIZE_RETAIL};
 const Info<std::string> MAIN_GFX_BACKEND{{System::Main, "Core", "GFXBackend"},
                                          VideoBackendBase::GetDefaultBackendName()};
+const Info<HSP::HSPDeviceType> MAIN_HSP_DEVICE{{System::Main, "Core", "HSPDevice"},
+                                               HSP::HSPDeviceType::HSPDEVICE_NONE};
+const Info<u32> MAIN_ARAM_EXPANSION_SIZE{{System::Main, "Core", "ARAMExpansionSize"}, 4194304};
```

Comment:
nit: use hex or some multiplies, or some named value, etc...to get rid of the magic base10

Reply:
Perhaps then the CustomRTCValue default should be changed to 10957 * 24 * 60 * 60 too.

Reply:
#10684